src/test/javascript/portal/ui/openlayers/control/SpatialConstraintSpec.js

```diff
@@ -125,6 +125,14 @@ describe('Portal.ui.openlayers.control.SpatialConstraint', function() {
             spatialConstraint.layer.events.triggerEvent('sketchcomplete', { feature: feature });
             expect(spatialConstraint.clear).not.toHaveBeenCalled();
         });
+
+        it("getCleanGeometry fixes Geometries with longitudes > 180 (possible Openlayers 1.13.1 behaviour using EPSG:4326) ", function() {
```

Comment:
No need for the comment in this line. It's OpenLayers 2.13.1, not 1.13.1 though.